fastestimator/op/numpyop/univariate/minmax.py

```diff
@@ -33,15 +33,21 @@ class Minmax(NumpyOp):
         ds_id: What dataset id(s) to execute this Op in. To execute regardless of ds_id, pass None. To execute in all
             ds_ids except for a particular one, you can pass an argument like "!ds1".
         epsilon: A small value to prevent numeric instability in the division.
+        new_min: The desired minimum value after the minmax operation.
+        new_max: The desired maximum value after the minmax operation.
     """
     def __init__(self,
                  inputs: Union[str, Iterable[str]],
                  outputs: Union[str, Iterable[str]],
                  mode: Union[None, str, Iterable[str]] = None,
                  ds_id: Union[None, str, Iterable[str]] = None,
-                 epsilon: float = 1e-7):
+                 epsilon: float = 1e-7,
+                 new_min: float = 0.0,
+                 new_max: float = 1.0):
         super().__init__(inputs=inputs, outputs=outputs, mode=mode, ds_id=ds_id)
         self.epsilon = epsilon
+        self.new_min = new_min
```

Comment:
should we have an assert for new_max > new_min?

Reply:
Added, good point.